### ISSUE TITLE
use alt attributes on the image tags

### ISSUE THREAD

**Issue by rimmesbe** [SophGiles/ColmarAcademy#3]
Try to get in the habit of describing your images with the alt attribute. This description will show up if your image fails to load, it will be read by screen readers for accessibility, and it makes your code look more professional. 
https://www.codecademy.com/articles/accessibility

https://github.com/SophGiles/ColmarAcademy/blob/master/index.html#L123
